addons/actions.md

```diff
@@ -23,62 +23,88 @@ The following Actions are a core part of openHAB and do not need to be separatel
+- `say(String text)`: Says the given text through Text-to-Speech
+- `say(String text, String voice)`: Says the given text through the given voice (depends on the TTS engine and voices installed)
+- ` say(String text, String voice, String device)`: Says the given text through the given voice to the given output device (Mac OSX only).
 
 To get a list of available voices use `say -v ?` and to get a list of devices uses `say -a ?`.
 
 ### Logging Actions
 
-* `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
-* `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
-* `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
-* `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
+- `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
+- `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
+- `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
+- `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
 
 `logText` can be a compete String, constructed through concatination, or through arguments.
 
-* **Complete String Example:** `logInfo("Garage", "This is a complete String")`
-* **Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
-* **Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
+- **Complete String Example:** `logInfo("Garage", "This is a complete String")`
+- **Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
+- **Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
 
 One can configure whether specific log entries are logged out and where they get logged to (e.g. to a separate file) by [editing the logger configuration]({{base}}/administration/logging.html).
 
+### Notification actions
+
+Notification actions may be placed in `rules` to send alerts to mobile devices registered with <https://myopenhab.org>.  Three different actions are available:
+
+- `sendNotification("your myopenHAB user email address here","message")`: Sends a notification containing 'message' to device(s) registered with your myopenHAB email address at <https://myopenhab.org>
+- `sendBroadcastNotification("message")`: Sends a notification containing "message" to *all* devices of *all* users of your myopenHAB.org account
+- `sendLogNotification("message")`: Sends a log notification containing "message" to the `notifications` list at myopenHAB.org.  Notifications are NOT sent to any registered devices
+
+**Example**
+
+The following example sends a notification once per minute to the myopenHAB app running on your mobile device.
+This rule can be useful when testing notification actions.
```

Comment:
Not sure if an example is needed, all other sections don't have one. If you don't want to remove it consider moving it to the top or add a section, example and link to this page in the rules-dsl article.

Reply:
Also I don't like the example as it is not really meaningful. Check other rules-dsl examples to bring it in line.

Reply:
@ThomDietrich Hum - not exactly sure what you would prefer here.  Speaking personally, having recently worked on notifications using OH Cloud, Prowl and others, this rule was EXTREMELY useful to me for troubleshooting (such a simple thing, but it really helped).  I am perfectly happy to remove it entirely, or to put it in rules-dsl, or to reformat it somehow, if you are saying it is not formatted correctly.  But for a new user, or even a user who has been using OH for a while but is just getting into notifications, this is a "breadcrumb" - a hint at a super-simple but useful troubleshooting tool.  Please let me know what you would like.

Reply:
bump...

Reply:
I agree - why provide an example here when the other sections do not have one.  I will delete it.

Reply:
I've looked at the Actions article again and think we should reconsider. How about you move this section out of "Core Actions"? Other than those, this section is only relevant for Cloud users and it is a notifications action which is quite different what core actions do.
I'd suggest to make it a top-level headline between "Core Actions" and "Installable Actions".

About the example: With the above I now also agree an example is a good idea. Sorry for the back and forth. What I didn't like about yours is, that it highlights how to troubleshoot openHAB. Ideally that is not needed. Hence my suggestion would be a production-like example. How about:


```javascript
rule "Front Door Notification"
when
    Item Apartment_FrontDoor changed to OPEN 
then
    send Notification("me@email.com", "Front door was opened!")
end
```